modules/ichorcna/1.0/ichorcna.smk

```diff
@@ -242,7 +256,7 @@ rule _run_ichorcna:
         scStates = CFG["options"]["run"]["ichorCNA_scStates"],
         maxCN = CFG["options"]["run"]["ichorCNA_maxCN"],
         includeHOMD = CFG["options"]["run"]["ichorCNA_includeHOMD"],
-        chrs = op.switch_on_wildcard("genome_build", CFG["options"]["run"]["ichorCNA_chrs"]),
+        chrs = get_chromosomes_R,
```

Comment:
Does this work?

Reply:
Yeah it prints out:  `--chrs "c('chr1','chr2','chr3','chr4','chr5','chr6','chr7','chr8','chr9','chr10','chr11','chr12','chr13','chr14','chr15','chr16','chr17','chr18','chr19','chr20','chr21','chr22','chrX','chrY')" ` in dry run